src/OpenLoco/src/Objects/WallObject.h

```diff
@@ -15,6 +16,20 @@ namespace OpenLoco
         struct RenderTarget;
     }
 
+    enum class WallObjectFlags : uint8_t
+    {
+        none = 0U,
+        unk0 = 1U << 0,
+        unk1 = 1U << 1,
+        onlyOnLevelLand = 1U << 2,
+        unk3 = 1U << 3,
+        unk4 = 1U << 4,
+        unk5 = 1U << 5,
+        unk6 = 1U << 6,
+        unk7 = 1U << 7,
```

Comment:
OpenRCT2:
```cpp
    WALL_SCENERY_HAS_PRIMARY_COLOUR = (1 << 0),   // 0x1
    WALL_SCENERY_HAS_GLASS = (1 << 1),            // 0x2
    WALL_SCENERY_CANT_BUILD_ON_SLOPE = (1 << 2),  // 0x4
    WALL_SCENERY_IS_DOUBLE_SIDED = (1 << 3),      // 0x8
    WALL_SCENERY_IS_DOOR = (1 << 4),              // 0x10
    WALL_SCENERY_LONG_DOOR_ANIMATION = (1 << 5),  // 0x20
    WALL_SCENERY_HAS_SECONDARY_COLOUR = (1 << 6), // 0x40
    WALL_SCENERY_HAS_TERTIARY_COLOUR = (1 << 7),  // 0x80
```

Reply:
Yeah, I'm seeing colour assignments in this PR, fortunately.

Reply:
Yes for whatever reason it can handle colour in the object but not in the ui guess it's left over from rct2

Reply:
Smells like a feature we can implement

Reply:
I think it would be nice to implement a more extensive scenery system eventually. Sandbox games have really shown players to get creative when provided the right tools. (Sadly, vanilla Locomotion even hid the walls tab outside of the editor...)